src/components/WalletAssetsHeadline.vue

```diff
@@ -0,0 +1,240 @@
+        </div>
+      </el-popover>
+    </div>
+    <s-divider class="wallet-assets-divider" />
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Mixins, Prop } from 'vue-property-decorator';
+
+import { mutation, state } from '@/store/decorators';
+import FormattedAmount from './FormattedAmount.vue';
+import TranslationMixin from './mixins/TranslationMixin';
+import LoadingMixin from './mixins/LoadingMixin';
+import { WalletAssetFilters, WalletFilteringOptions } from '../consts';
+
+@Component({
+  components: {
+    FormattedAmount,
+  },
+})
+export default class WalletAssetsHeadline extends Mixins(TranslationMixin, LoadingMixin) {
+  @Prop({ default: '0', type: String }) readonly assetsFiatAmount!: string;
+
+  @state.settings.filters filters!: WalletAssetFilters;
+  @mutation.settings.setFilterOptions private setFilterOptions!: (filter: WalletAssetFilters) => void;
+
+  verifiedOnlySwitch = false;
+  zeroBalanceSwitch = false;
+
+  get onlyVerifiedAssets(): boolean {
+    return this.filters.verifiedOnly;
+  }
+
+  set onlyVerifiedAssets(value: boolean) {
+    this.updateFilters('verifiedOnly', value);
+  }
+
+  get zeroBalanceAssets(): boolean {
+    return this.filters.zeroBalance;
+  }
+
+  set zeroBalanceAssets(value: boolean) {
+    this.updateFilters('zeroBalance', value);
+  }
+
+  get selectedFilter(): string {
+    return this.filters.option;
+  }
+
+  set selectedFilter(value) {
+    this.updateFilters('option', value);
+  }
+
+  updateFilters(key: string, value: string | boolean): void {
+    const filters = {
+      ...this.filters,
+      [key]: value,
+    };
+
+    this.setFilterOptions(filters);
+    this.$emit('filter-assets', filters);
+  }
+
+  get filterOptions(): Array<string> {
+    return Object.values(WalletFilteringOptions);
+  }
+
+  updated(): void {
+    if (this.filters.option === WalletFilteringOptions.NFT) {
+      // disable verified only switch as there are no whitelisted NFTs.
+      this.verifiedOnlySwitch = true;
+      if (this.onlyVerifiedAssets === true) {
+        this.onlyVerifiedAssets = false;
+      }
+    } else {
+      this.verifiedOnlySwitch = false;
+    }
+  }
+
+  get chosenOptionText(): string {
+    switch (this.filters.option) {
+      case WalletFilteringOptions.ALL:
+        return this.t('filter.all');
+      case WalletFilteringOptions.TOKEN:
+        return this.t('filter.token');
+      case WalletFilteringOptions.NFT:
+        return this.t('filter.nft');
+      default:
+        return this.t('filter.all');
+    }
+  }
+
+  mounted(): void {
+    this.onlyVerifiedAssets = this.filters.verifiedOnly;
+    this.zeroBalanceAssets = this.filters.zeroBalance;
+    this.selectedFilter = this.filters.option;
+    this.$emit('filter-assets', this.filters);
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+.el-radio {
+  font-weight: 300;
+}
+.el-popover {
+  padding: 0 !important;
+}
+</style>
+
+<style lang="scss">
+$size-px: 16px;
+
```

Comment:
In current styles area we have a lot of px values, is it possible to use variables for them or all of them are unique?

Reply:
they're unique, these sizes look good in markup, others are not so concise